vue/components/SearchPageComponents/searchResultOutline.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div class="col commentContent">
+          <p class="text-muted"><slot name="typeLabel">type of search result here</slot></p>
+          <p class="mb-2" style="font-size:1.3em">
+              <img class="postAuthorImage" v-bind:src="image" />
+              <slot name="title">search result title goes here</slot>
```

Comment:
I haven't used Vue slots before, but I take it that this whole tag gets replaced with the prop passed in?

Reply:
Yes, except the `<slot>` tag gets replaced by a `<template>` contribued by the component that uses and renders the `searchResultOutline` component, not a prop. (The "goes here" text will show up as a placeholder if the component that renders `searchResultOutline` fails to and over a `<template>` with the `v-slot:title` attribute, although that's not intended to ever actually happen)